include/boost/url/params.hpp

```diff
@@ -338,6 +351,11 @@ class params
     bool
     empty() const noexcept;
 
+    /** Determine the container size
```

Comment:
"Return the number of elements"

Reply:
The briefs for functions which return values usually start with the word "Return."

Reply:
This isn't resolved, it should read:
```
/** Return the number of elements
...
```

Reply:
This preview is outdated.

https://github.com/alandefreitas/url/blob/9594ab9b7c0e9151440a1310cffbfe9806fe55d7/include/boost/url/params.hpp#L349